src/plugins/page-icons/page-icons.js

```
@@ -0,0 +1,78 @@
+					return null
+				}
+
+				const parsed = parseSVGString( icon )
+				if ( ! parsed ) {
+					return null
+				}
+
+				const { attributes: svgAttributes, innerHTML } = parsed
+
+				return (
+					<symbol key={ iconId } id={ `stk-page-icons__${ iconId }` } { ...svgAttributes } dangerouslySetInnerHTML={ { __html: innerHTML } } />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)